TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/POST-Negative-Deviceresource.robot

```diff
@@ -0,0 +1,82 @@
+*** Settings ***
```

Comment:
Add `[Teardown]` to delete device profile to the testcases containing `Create A Device Profile Sample`.